src/Compilers/VisualBasic/Portable/Scanner/CharacterInfo.vb

```diff
@@ -16,7 +16,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
     ''' <summary>
     ''' Provides members for determining Syntax facts about characters and Unicode conversions.
     ''' </summary>
+#Disable Warning CA1052 ' Static holder types should be Static or NotInheritable - Public API
     Partial Public Class SyntaxFacts
+#Enable Warning CA1052 ' Static holder types should be Static or NotInheritable
```

Comment:
we should still make this static.  no one can/should be instantiating this.  and if they are, i think they should break.

Reply:
@CyrusNajmabadi That is exactly what I was asking in https://github.com/dotnet/roslyn/pull/65830#issuecomment-1340713611.

Reply:
Should I open an issue to track this?